modules/Bio/EnsEMBL/Variation/BaseVariationFeatureOverlapAllele.pm

```diff
@@ -265,7 +265,7 @@ sub get_all_OverlapConsequences {
     OC: for my $oc (@oc_list) {
 
       last if $assigned_tier && $oc->{tier} > $assigned_tier;
-      my $shifting_offset = defined($self->{shift_hash}) ? $self->{shift_hash}->{shift_length} * $tr->strand : 0;
+      my $shifting_offset = $bvfo->{shifted} && defined($self->{shift_hash}) ? $self->{shift_hash}->{shift_length} * $tr->strand : 0;
```

Comment:
A comment about where shifted is set would be useful. For ex `shifted is set in TranscriptVariation`.